### DISCUSSION TITLE
Renovate adds wrong architecture in Yarn Cache

### DISCUSSION THREAD

**Discussion by twiesing** [renovatebot/renovate#32118] [Request Help]
### How are you running Renovate?

Self-hosted Renovate

### Please tell us more about your question or problem

My GitLab runner for CI (where Renovate is executed in a pipeline with Docker) runs on the amd64 platform. My target system is an arm64 host. Unfortunately, Renovate updates the yarn cache with the wrong architecture.

Any ideas?

<img width="1233" alt="Bildschirmfoto 2024-10-24 um 11 26 34" src="https://github.com/user-attachments/assets/7c63d9b8-551c-4c27-9e48-941adc91d1bd">

**Comment by rarkins** [comment]
Renovate doesn't do this directly - it's Yarn itself making updates to this cache.  Your question is really one for Yarn itself instead of Renovate. Can you run Yarn on amd64 while caching arm64 packages?

**Comment by viceice** [comment]
https://yarnpkg.com/configuration/yarnrc#supportedArchitectures
